common/views/components/Buttons/ButtonButton/README.md

```diff
@@ -0,0 +1,10 @@
+### Primary button
+Main call to action on the page, e.g. 'Booking' or 'Sign up' - may include an icon.
+
+### Secondary button
+Used alongside the primary button as a supporting action, e.g. 'Cancel'.
+
+### Tertiary button
+Used to define multiple actions which should be separated from page content.
+
+These can be called either as a `button` or an `a` element, using the `<ButtonButton />` or `<LinkButton />` components.
```

Comment:
Do we every use the ButtonButton?
I think we only use links?

Reply:
Are you asking specifically about the tertiary buttons? If so, I think you're right – so far we only implement them as `LinkButton`s (but nothing's stopping us from using `ButtonButton`s in the future, as the styles come along for the ride.

Reply:
> (but nothing's stopping us from using ButtonButtons in the future

Complexity of the design system should.